Python/frozen_hello.h

```diff
@@ -1,13 +1,12 @@
-/* Generated with Tools/freeze/regen_frozen.py */
-static unsigned char M___hello__[] = {
-    227,0,0,0,0,0,0,0,0,0,0,0,0,
-    0,0,0,0,2,0,0,0,64,0,0,0,115,
-    16,0,0,0,100,0,90,0,101,1,100,1,131,
-    1,1,0,100,2,83,0,41,3,84,122,12,72,
-    101,108,108,111,32,119,111,114,108,100,33,78,41,
-    2,90,11,105,110,105,116,105,97,108,105,122,101,
-    100,218,5,112,114,105,110,116,169,0,114,2,0,
-    0,0,114,2,0,0,0,218,4,110,111,110,101,
-    218,8,60,109,111,100,117,108,101,62,1,0,0,
-    0,115,2,0,0,0,4,1,
+/* Auto-generated by Programs/_freeze_importlib.c */
+const unsigned char _Py_M__hello[] = {
+    99,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+    0,2,0,0,0,64,0,0,0,115,16,0,0,0,100,0,
+    90,0,101,1,100,1,131,1,1,0,100,2,83,0,41,3,
+    84,122,12,72,101,108,108,111,32,119,111,114,108,100,33,78,
+    41,2,90,11,105,110,105,116,105,97,108,105,122,101,100,218,
+    5,112,114,105,110,116,169,0,114,1,0,0,0,114,1,0,
+    0,0,122,14,60,102,114,111,122,101,110,32,104,101,108,108,
+    111,62,218,8,60,109,111,100,117,108,101,62,1,0,0,0,
+    115,6,0,0,0,4,0,12,1,255,128,
 };
```

Comment:
I don't understand why the file is suddenly so different when in the previous pull requests, only one line was updated.

Reply:
I think it's because the co_filename has change.  _freeze_importlib sets it to "<frozen hello>".  Previously it was set to "none".

Reply:
Oh, and _freeze_importlib line wraps the byte array a bit differently.